packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -284,9 +339,32 @@ export class PendingStateManager {
         return nextPendingState;
     }
 
+    public replayInitialStates() {
```

Comment:
I feel like this method should not exists, if we can use this.pendingStates instead of adding this.pendingRebase
By the time we get connected, we need to ensure that we have observed all the ops that are baselines (referenceSequenceNumber) for all locally saved ops and they were converted to exactly same state (across DDSs and pending Manager) as if they were regular local ops. And then existing processes could take care of the rest.
The least amount of special code we have for this state - the better, as it will be easier to reason about logic and less likely we will make mistakes.